c/consul/Dockerfile/1.9.4-ubi8/Dockerfile.txt

```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8
 
 # This is the release of Consul to pull in.
-ENV CONSUL_VERSION=v1.9.4
+ARG CONSUL_VERSION
```

Comment:
Can you provide default value for `CONSUL_VERSION` as `v1.9.4` ?
So that, if no build argument provided, this will take the default value.